### ISSUE TITLE
1.7.10 fml invalid payload disconnect.

### ISSUE THREAD

**Issue by popcorn9499** [HexagonMC/BungeeCord#162]
>bungee
11:26:34 [INFO] This server is running popicraft version git:BungeeCord-Bootstrap:1.12-SNAPSHOT:53fda71:154 by md_5
11:26:34 [INFO] Protocol support for 1.7.x by Zartec, ghac and I9hdkill

### Are you using Bungee plugins? If yes, please list all of them + versionnumber
I actually removed all plugins from bungee and it still occured.

### How can we reproduce the error/bug? 
Im still working on this. I have found that i get the invalid payload when i get pushed to the hub server and it drops me there and it only happens with the mods on my client any other forge client seems to work yet this just recently started to do this. the client worked fine before. 

The forge server is running 1.7.10 fml version 10.13.4.1614
and the spigot hub is running 1.11.2 with protocol support and via version. There are some other plugins and I'm going to look to see which plugin is the problem.

I'm going to go from the ground up to find the cause of which mod if any and find out what is actually causing my issue here. I just figured I'd post and see if anyone else has seen this issue. It only recently started and I'm looking at what changed myself. 

Thanks for reading i will provide more information as i find it. Sorry for such a vague post.

**Comment by spannerman79** [comment]
Possible dupe #152

**Comment by popcorn9499** [comment]
I just noticed. my bungee log has a long string of stuff at login but i never rlly paid much attention to it as it worked before. Its just the fml handshake till it reachs a point then it throws an exception but exceptions have always shown up but the game connection works fine. https://pastebin.com/4DqrKGRN thats the log. I'm going to test shortly to see if downgrading my bungee to the version specified works.

**Comment by popcorn9499** [comment]
downgrading to bungee v135 worked. Ill get back to u when i figure out which mod or mods are causing it

**Comment by popcorn9499** [comment]
only thing i find strange about all this is the other invalid payload thingy didnt seem to be mentioned there. So either this has a slightly different issue or there is more than just that to this problem? I'll continue to work on this in abit.

**Comment by popcorn9499** [comment]
Its looking like a dup of the other issue 152 sorry. anyways ill post in there what i find to be the problemed mods in the pack that ive got. Just weird that when i introduced my entire pack i got a invalid payload on connection to the vanilla hub

**Comment by popcorn9499** [comment]
I found the problem only occured with the cofh mods and that i couldnt reproduce the invalid payload issue when it pushed me to the vanila hub server. But i did find that running this version worked as well as anything below it seemed to work.
 `15:52:49 [INFO] This server is running popicraft version git:BungeeCord-Bootstrap:1.11-SNAPSHOT:faccbda:140 by md_5
15:52:49 [INFO] Protocol support for 1.7.x by Zartec, ghac and I9hdkil`

I tried release v135 and v140 and both worked anything above as far as ik right now doesnt work. Im positive v150 and above doesnt work however.
